lib/parser.js

```
@@ -9,6 +9,11 @@ var Parser = module.exports = function(markup, opts) {
   opts = opts || {}
 
   this.markup = $(markup)
+
+  if (this.markup.length > 1) {
```

I don't think we should raise an error for 0, at least not this error, since it would be... misleading, as 0 is not multiple roots.  It seems more likely to cause problems/confusion.  If 0 is invalid AND causes a problem, I think it should be addressed separately.  If it's invalid but works fine, or invalid but never occurs in practice, I think >1 is more intention revealing anyway.